README.md

```diff
@@ -3,5 +3,3 @@
 ## Menelaos, Urselaan
 
 A website where you make anonymous posts and takes you their path in the db
```

Comment:
The README is quite empty. Add a bit more info

Reply:
just like your heart